src/deepsparse/server/cli.py

```diff
@@ -156,6 +156,15 @@ def config(config_path: str, host: str, port: int, log_level: str):
     default="info",
     help="Sets the logging level.",
 )
+@click.option(
+    "--no-loggers",
+    type=bool,
```

Comment:
```suggestion
    is_flag=True,
```
https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags